examples/utils/webpack-fluid-loader/src/routes.ts

```diff
@@ -70,7 +70,11 @@ export const after = (
 	baseDir: string,
 	env: Partial<RouteOptions>,
 ) => {
-	const options: RouteOptions = { mode: "local", ...env, ...{ port: server.options.port } };
+	const options: RouteOptions = {
+		mode: "local",
+		...env,
+		...{ port: server.options.port ?? 8080 },
+	};
```

Comment:
Seems like there were some typing updates that allow undefined. Coalescing to the default port was the most expedient way forward.